types/types.ts

```diff
@@ -66,3 +66,22 @@ export interface QueryInsightsSettingsResponse {
   group_by?: string;
   exporter?: ExporterSettingsResponse;
 }
+
+export interface LiveSearchQueryRecord {
```

Comment:
Wondering if we will need some of the other attributes from SearchQueryRecord her in the future if not now:
```
export interface SearchQueryRecord {
  timestamp: number;
  measurements: {
    latency?: Measurement;
    cpu?: Measurement;
    memory?: Measurement;
  };
  total_shards: number;
  node_id: string;
  source: ISearchSource;
  labels: Record<string, string>;
  search_type: string;
  indices: string[];
  phase_latency_map: PhaseLatencyMap;
  task_resource_usages: Task[];
  id: string;
  group_by: string;
}
```